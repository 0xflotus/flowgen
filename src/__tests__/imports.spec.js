import { compiler, beautify } from "..";

it("should handle imports", () => {
  const ts = `import { GeneratorOptions } from "@babel/generator";
import traverse, { Visitor, NodePath } from "@babel/traverse";
import template from "@babel/template";
import * as t from "@babel/types";
import v, * as d from 'typescript';`;
  const result = compiler.compileDefinitionString(ts);
  expect(beautify(result)).toMatchSnapshot();
});
