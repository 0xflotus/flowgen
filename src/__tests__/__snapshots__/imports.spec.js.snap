// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle imports 1`] = `
"import type { GeneratorOptions } from \\"@babel/generator\\";
import traverse, { Visitor, NodePath } from \\"@babel/traverse\\";
import template from \\"@babel/template\\";
import * as t from \\"@babel/types\\";
import v, * as d from \\"typescript\\";
"
`;
